tests/migration/nodeselector.go

```diff
@@ -0,0 +1,252 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package migration
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/cleanup"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libmigration"
+	"kubevirt.io/kubevirt/tests/libnet"
+	"kubevirt.io/kubevirt/tests/libnode"
+	"kubevirt.io/kubevirt/tests/libpod"
+	"kubevirt.io/kubevirt/tests/libvmifact"
+	"kubevirt.io/kubevirt/tests/libwait"
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+var _ = SIGMigrationDescribe("Live Migration", decorators.RequiresThreeSchedulableNodes, func() {
+	var virtClient kubecli.KubevirtClient
+	var nodes *k8sv1.NodeList
+
+	BeforeEach(func() {
+		virtClient = kubevirt.Client()
+		Eventually(func() int {
+			nodes = libnode.GetAllSchedulableNodes(virtClient)
+			return len(nodes.Items)
+		}, 60*time.Second, 1*time.Second).Should(BeNumerically(">=", 3), "There should be at lest three compute nodes")
+	})
+
+	Context("configuring AddedNodeSelector on migration", Serial, func() {
```

Comment:
To avoid confusion, it might make sense to put the `Serial` decorator into `SIGMigrationDescribe`. You might also be able to drop this extra context.

Reply:
done